native/PluginProcessor.cpp

```diff
@@ -262,9 +262,23 @@ void EffectsPluginProcessor::initJavaScriptEngine()
         return choc::value::Value();
     });
 
-    jsContext.registerFunction("__log__", [](choc::javascript::ArgumentList args) {
+    jsContext.registerFunction("__log__", [&](choc::javascript::ArgumentList args) {
```

Comment:
tiny nit-pick, but let's change the capture group to `[this]`. IMO it makes it more clear that we're going to be accessing the current object in the body of the func

Reply:
Great! - my understanding is very limited here :)